src/Uno.UWP/Devices/Bluetooth/BluetoothLEDevice.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.Devices.Bluetooth
+{
+	public partial class BluetoothLEDevice : global::System.IDisposable
+	{
+		private static string _deviceSelectorPrefix = "System.Devices.DevObjectType:=5 AND System.Devices.Aep.ProtocolId:=\"{BB7BB05E-5972-42B5-94FC-76EAA7084D49}\" AND ";

```

Comment:
Are the constant strings for this class coming from the execution on Windows?

Reply:
Yes, all selectors are from standard UWP - I created simple app to get all possible BT selectors, and copied these selectors.